src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(callback, number) {
+      this.result = callback(number, this.result);
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(number, currentResult) {
+      return currentResult + number;
```

Comment:
Actually you don't need to use `currentResult ` as a second param in your method. You should use access to your result property via `this`